### PR TITLE
fix dishes scraping path

### LINKED ISSUE #25: Munich Student Union Side Dish parsing
Currently, we do not parse side dishes for the Munich Student Union canteens.
For this we have to do one request for each day and the parse the result.

1. Get an overview about all available days from: https://www.studentenwerk-muenchen.de/mensa/speiseplan/speiseplan_422_-de.html
2. Request the extended menu for each day: https://www.studentenwerk-muenchen.de/mensa/speiseplan/speiseplan_2021-09-13_422_-de.html

### LINKED ISSUE #251: Menu fetching doesn't work
https://www.studierendenwerk-muenchen-oberbayern.de/mensa/speiseplan/speiseplan_422_-de.html shows dishes, but [the API](https://github.com/TUM-Dev/eat-api/blob/gh-pages/mensa-garching/2024/30.json) returns none.

### PR DESCRIPTION
**Pull request by Friendly-Banana**
fix linter issues
fixes #251

### CODE REVIEW

**Comment by CommanderStorm** [comment]
Between linter changes and actual changes this is pretty hard to review.
Could you please split them into different PRs or commits?

What did you do to achieve the linting difference?

**Comment by Friendly-Banana** [comment]
I added 3 spaces to convert `:` into `: ` in `src/entities.py`. Should I move this into a separate commit?
All other changes are simply replacing the old class `js-schedule-dish-description` with the new `c-menu-dish__title`, first to adapt the scraper, then to fix the tests.

**Comment by Friendly-Banana** [comment]
Split up into separate commits.

File: src/test/assets/studentenwerk/mensa-arcisstr/for-generation/2021-09-13.html [hunk 1]
```diff
@@ -510,7 +510,7 @@ <h1 class="c-page__title  u-color-current">
...
+				</span><p class="c-menu-dish__title">Ravioli mit Pilzfüllung in pikanter Kräutersauce <span class="c-schedule__marker  c-schedule__marker--type  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen=""><span class="u-text-sup"></span></span><span class="c-schedule__marker  c-schedule__marker--additional  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen=""><span class="u-text-sup"></span></span><span class="c-schedule__marker  c-schedule__marker--allergen  [ c-tooltip  c-tooltip--inverse  c-tooltip--schedule ]  js-schedule-tooltip" data-essen="Ei,Gl,GlW,Kn,Mi,Sl"><span class="u-text-sup">[Ei,Gl,GlW,Kn,Mi,Sl]</span></span></p></dd></dl></li>
```
Comment (CommanderStorm) [inline src/test/assets/studentenwerk/mensa-arcisstr/for-generation/2021-09-13.html:513]: how did you genereate these changes?
Did you re-download those html pages?

If yes, could you bump the timestamp in the filename?
If no, could you re-download newer versions of that page?

**Comment by CommanderStorm** [comment]
> Between linter changes and actual changes this is pretty hard to review.

Sorry, was mislead by the changed line count that this is mostly linting ^^

File: src/test/assets/studentenwerk/mensa-arcisstr/for-generation/2021-09-13.html (hunk 1 above)
Comment (Friendly-Banana) [reply src/test/assets/studentenwerk/mensa-arcisstr/for-generation/2021-09-13.html]: I originally only replaced one xapth, as that was enough for the other commenter on #251. I now downloaded all pages and fixed the other xpaths.

**Review by CommanderStorm — APPROVED** [review]
LGTM, passes tests (have adjusted CI to also run on PRs the next time ^^).
Thanks for the contribution ❤️

**Pull request merged**